styles/base/inputs/_input-container.scss

```diff
@@ -411,9 +404,10 @@
   .sprk-b-TextInputIconContainer
   .sprk-b-TextInput:focus
   + .sprk-b-Label {
-  padding: $sprk-text-input-huge-has-text-icon-padding;
+  padding: $sprk-input-huge-has-icon-padding;
 }
```

Comment:
Did we have a typo here before that we changed it to this?

Reply:
@guern1kn worked on this file